### DISCUSSION TITLE
Jailbreak v5 for Control4 3.2.3

### DISCUSSION THREAD

**Discussion by garrynewman** [garrynewman/Control4.Jailbreak#21] [General]
The same as v4 except

- Works properly with 3.2.3

<hr /><em>This discussion was created from the release <a href='https://github.com/garrynewman/Control4.Jailbreak/releases/tag/v5'>Jailbreak v5 for Control4 3.2.3</a>.</em>

**Comment by GeniusXYZZY** [comment]
Hi Garry.... Do you have a version for OS 2.10? I am keen to get my composer up and running again as having just retired, I have lost my dealer registration ( was a dealer for many years) and I want to manage my private system till I croak. Any help would be greatly appreciated - thank you :)

**Reply by bob657044** [reply]
Hi Gary
will this work with latest C4 3.4?
Cheers

**Reply by cgbale** [reply]
Yes

**Comment by avig2023** [comment]
Does it work with 3.3.2 ?

**Reply by ketenn57** [reply]
Just upgraded and it doesn't seem to.

**Reply by cgbale** [reply]
It works with 3.3.2 just fine. Same a prior releases.

**Reply by altecsysaz** [reply]
can you help with why i get a "Permission denied (password)." error when i try to patch the director certificate then?

**Reply by DanITman** [reply]
Its because the password is not being calculated properly.  This version does not properly calculate the password when on a VPN or running in virtual.  Run it in windows on the same network and the password will calculate properly.   Another thing you can try is to see if the default password works "t0talc0ntr0l4!"  This will work when there isn't a project on the controller.

**Reply by michaeldoddgit** [reply]
you can do a Network/Password reset by holding down the ID button while powering up for a minute.  Then just type into the jailbreak root / t0talc0ntr0l4!

This will not erase your project like holding the reset button down would

**Comment by avig2023** [comment]
Thank you so much for confirming that it works. I keep getting an error
“permission denied" when I try to patch the director.

**Reply by altecsysaz** [reply]
i get the same error on 3.3.2. When i try to patch the director certificate i get: Permission denied (password). in the LogWindow that pops up. not sure what to do now...

**Reply by avig2023** [reply]
Thank you. Unfortunately, I don't have a windows machine so I'm running it on a MAC with UTM with Windows 11 on.

**Reply by DanITman** [reply]
I can put your password here, let me know what your mac address is.  If you aren't comfortable with that. Let me know how I can contact you.

**Reply by avig2023** [reply]
> I can put your password here, let me know what your mac address is. If you aren't comfortable with that. Let me know how I can contact you.
[DanITman](https://github.com/DanITman), I'm not siure if you got my email address. as it is blocked here. How can I provide yo with my email address?

**Comment by avig2023** [comment]
You can email me at ***@***.***
I'm not sure how to privately message you here...

**Reply by DanITman** [reply]
It won't let you.  You can reach me on Discord under the same username.

**Reply by avig2023** [reply]
> It won't let you. You can reach me on Discord under the same username.
I sent you a friend request on Discord.

**Comment by KONEOX** [comment]
hi bro 
I have question regarding Jailbreak ,I can not connect with director please any  advise or method can I do .thanks in advance
